.eslintrc.js

```diff
@@ -24,12 +24,13 @@ module.exports = {
   },
   rules: {
     "one-var": ["error", "never"],
+    "no-unused-vars": ["error", { argsIgnorePattern: "^_" }],
```

Comment:
Only something in deployer, could also be in-line ignored

Reply:
I don't understand what `{ argsIgnorePattern: "^_" }` means. I'd have to comment this out, run `yarn eslint` and then perhaps I'd understand what this does and what effect it has. 
So if this line is critical, because it's so "odd", it needs to be explained with a code comment for the future-us.

Reply:
The is something buried inside deployer using an unused parameter `_content`. It sounds like it's probably better to in-line ignore if this isn't a "patter" across the repo you use

Reply:
In fact, why do we even have that second argument there @fiji-flo ??
https://github.com/mdn/yari/blob/3a200d60bb12d6b388a2e381d9c5e11cb8215ab9/deployer/aws-lambda/content-origin-request/index.js#L46

Can we just delete that `, _context)` second argument?

Reply:
Yes, I prefixed it instead of deleting. That was my fist PR in this repo 🙈 

We can just delete it.

Reply:
Done